libraries/botbuilder-dialogs-adaptive/src/actions/sendList.ts

```diff
@@ -45,7 +45,7 @@ export class SendList extends Dialog {
     }
 
     protected onComputeId(): string {
-        return `SendList[]`;
+        return `SendList[${this.listProperty}]`;
```

Comment:
Should we remove this action? C# don't have this, this functionality can be done through LG.

Reply:
We can remove it assuming the LG pieces are working.

---
In reply to: [354679140](https://github.com/microsoft/botbuilder-js/pull/1476#discussion_r354679140) [](ancestors = 354679140)